src/mono/wasm/runtime/zlib-helper.c

```diff
@@ -0,0 +1,373 @@
+/*
```

Comment:
Why is it needed for net5?

Reply:
Will remove the needless files later when doing the final sync from mono/mono.

Reply:
Why is it needed for net5?